src/byteArrayToString.ts

```diff
@@ -0,0 +1,3 @@
+export function byteArrayToString(value: Uint8Array) {
```

Comment:
it feels a bit odd that this is in a separate file whereas `stringToByteArray` is in utils